modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -1,47 +1,105 @@
-= Anypoint Custom Metrics Connector
+= Custom Metrics
 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using a custom dashboard in advanced mode and the built-in *Custom metrics* dashboard. 
+The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring to visualize metrics in real time using a custom dashboard in advanced mode and the built-in *Custom metrics* dashboard. 
 
-Once you define metrics in the Custom Metrics connector, you can select which metrics to display in the dashboard. You can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
+Define metrics in the Custom Metrics connector to select which metrics to display in the dashboard. You can filter and group the metrics along the included dimensions, and you can include multiple custom and built-in metrics in a single dashboard visualization.
```

Comment:
Are there both built-in dimensions and built-in metrics? Are there not custom dimensions? If not, maybe break into two sentences for clarity? And call things built-in everywhere (or default, or MuleSoft-defined or whatever term is used most often in this doc set).